packages/blade/docs/guides/Installation.stories.mdx

```diff
@@ -40,10 +40,14 @@ Before you install the package, make sure that you have performed following step
 
 ## ⬇ Add blade to your application
 
-1. Install blade as a dependency. Blade has a peer dependency on `styled-components`, you can skip adding it if you already have it installed in your project.
+1. Install blade as a dependency. 
+Blade has a peer dependency on few libraries, you can skip adding it if you already have it installed in your project.
+
+  - `styled-components`
+  - `@floating-ui/react`
```

Comment:
lets add that gorhom as well?

Reply:
That's for react-native I've already added it on the react-native section installation section. See 3rd point.